releasenotes/notes/0.10/0-10-release-8c37dadcc1c82fcc.yaml

```diff
@@ -72,29 +72,28 @@ features:
 upgrade:
   - |
     The return type of several save instructions have been changed to be the
-    corresponding `qiskit-terra` classes rather than raw numpy arrays or
+    corresponding Qiskit Terra classes rather than raw NumPy arrays or
     dictionaries. The types that have changed are
 
-    * :func:`~qiskit.providers.aer.libray.save_statvector` now returns as a
-      :class:`qiskit.quantum_info.Statevector`
-    * :func:`~qiskit.providers.aer.libray.save_density_matrix` now returns as a
-      :class:`qiskit.quantum_info.DensityMatrix`
-    * :func:`~qiskit.providers.aer.libray.save_stabilizer` now returns as
-      :class:`qiskit.quantum_info.StabilizerState`
-    * :func:`~qiskit.providers.aer.libray.save_unitary` now returns as
-      :class:`qiskit.quantum_info.Operator`
-    * :func:`~qiskit.providers.aer.libray.save_superop` now returns as
-      :class:`qiskit.quantum_info.SuperOp`
-    * :func:`~qiskit.providers.aer.libray.save_probabilities_dict`` now returns as a
-      :class:`qiskit.result.ProbDistribution`
+    * :func:`.save_statevector` now returns as a
+      :class:`~qiskit.quantum_info.Statevector`
+    * :func:`.save_density_matrix` now returns as a
+      :class:`~qiskit.quantum_info.DensityMatrix`
+    * :func:`.save_stabilizer` now returns as
+      :class:`~qiskit.quantum_info.StabilizerState`
+    * :func:`.save_unitary` now returns as
+      :class:`~qiskit.quantum_info.Operator`
+    * :func:`.save_superop` now returns as
+      :class:`~qiskit.quantum_info.SuperOp`
+    * :func:`.save_probabilities_dict` now returns as a
+      :class:`~qiskit.result.ProbDistribution`
 deprecations:
   - |
-    Using Numpy ``ndarray`` methods and attributes on the return type of
-    :func:`~qiskit.providers.aer.libray.save_statvector`,
-    :func:`~qiskit.providers.aer.libray.save_density_matrix`,
-    :func:`~qiskit.providers.aer.libray.save_unitary`,
-    :func:`~qiskit.providers.aer.libray.save_superop`, has been deprecated.
+    Using NumPy ``ndarray`` methods and attributes on the return type of
+    :func:`.save_statevector`, :func:`.save_density_matrix`,
```

Comment:
Same for these

Reply:
I built the whole docs locally and checked every Aer-internal link.